apps/backport/backport.py

```diff
@@ -275,6 +275,7 @@ def _upload_data_metadata(lg: Any, backport_short_name: str, dry_run: bool) -> N
             display={},
             dimensions=None,
         )
+        db_var.id = db_variable_row["id"]
```

Comment:
is it not possible to pass `id` as an argument of `gm.Variable`?

Reply:
This is the tricky part with SQLAlchemy ORM and dataclasses. I had to define that field as
```
id: Mapped[int] = mapped_column(Integer, primary_key=True, init=False)
```
with `init=False`, otherwise it would always require `id` when creating the class (which is nonsense because `id` gets autoincremented after inserting to DB). This is a special case, in most cases you shouldn't be defining `id` manually, either you create a new object without `id` or you get one from DB which already has one.

That said, it's gonna take some time to get used to SQLAlchemy architecture.